server/routes/bookedtoday/unexpectedArrivals/arrivals/validation/searchForExistingRecordsValidation.ts

```diff
@@ -0,0 +1,7 @@
+import { ValidationError, Validator } from '../../../../../middleware/validationMiddleware'
+
+const SearchForExistingRecordsValidator: Validator = (): ValidationError[] => {
+  return []
```

Comment:
validation to be done when later in sep tecket

Reply:
Probably best to remove validation code while it's not doing anything and add it when validation functionality is implemented in a later PR

Reply:
agreed